interfaces/cython/cantera/thermo.pyx

```diff
@@ -379,6 +379,22 @@ cdef class ThermoPhase(_SolutionBase):
         :param loglevel:
             Set to a value > 0 to write diagnostic output.
             """
+        # check for old parameters and warn the user
+        if 'maxsteps' in **kwargs:
+            max_steps = maxsteps
+            warnings.warn("To be deprecated after Cantera 2.5."
+                          "Use property `max_steps` instead", DeprecationWarning)
+
+        if 'maxiter' in **kwargs:
+            max_iter = maxiter
+            warnings.warn("To be deprecated after Cantera 2.5."
+                          "Use property `max_iter` instead", DeprecationWarning)
+
+        if 'loglevel' in **kwargs:
+            log_level = log_level
+            warnings.warn("To be deprecated after Cantera 2.5."
+                          "Use property `log_level` instead", DeprecationWarning)
```

Comment:
No need to include the backticks, this output will be shown unformatted anyways. I think the text of the warning message should say:

```
Keyword argument 'loglevel' is deprecated and will be removed after Cantera 2.5.0. Use argument 'log_level' instead.
```

And similar for the other warnings.